pkg/swarm/swarm.go

```diff
@@ -115,13 +120,17 @@ type Chunk interface {
 	TagID() uint32
 	WithTagID(t uint32) Chunk
 	Equal(Chunk) bool
+	Valid(...Validator) (bool, uint8)
```

Comment:
[another one bites the dust](https://github.com/ethersphere/bee/pull/602/files#diff-3e985044e26e3fa3cad69ba9df6cbeff290b80e6577d1a4f3ad169e100b7a59bL124). Didn't we just get rid of this stuff?

Reply:
true, yet, you see we need it..